### ISSUE TITLE
Project Tags: curated vs freeform options?

### ISSUE THREAD

**Issue by IanLee1521** [presidential-innovation-fellows/code-gov-web#195]
Has there been any thought about whether tags should be a curated list of possibilities (maybe something like the [Classifiers list](https://pypi.python.org/pypi?%3Aaction=list_classifiers) that is in, e.g., the Python community... vs allowing free form, anything goes tags that any project might define?

I think I would prefer the former, which could avoid issues where tags differ in spelling, but are equivalent in meaning, for instance: `svn` vs `subversion`

**Comment by okamanda** [comment]
Hi @IanLee1521, We're indeed looking at creating a meaningful taxonomy, but we're not quite there yet.

**Comment by IanLee1521** [comment]
@okamanda -- Great to hear!  Would it make sense to start a draft content page with even just a minimal list of tags, which can then live and evolve over time?

**Comment by okamanda** [comment]
@IanLee1521 Absolutely. What we're realizing is that the right set of tags or categories should help describe the function of the code (e.g., CMS, user authentication), in addition to the language it was written (e.g, Ruby) in or the topical area of its use (e.g, GSA New Hire App).

**Comment by IanLee1521** [comment]
A related question... Would it make sense for `tags` to be an optional rather than required field while the taxonomy is being developed?

**Comment by mattbailey0** [comment]
@IanLee1521 that last point is reasonable but making it required has helped us understand different possibilities for ways that agencies might use the field. It's really diverse!

**Comment by IanLee1521** [comment]
Absolutely! Part of my interest in this is not wanting to really tackle that diversity myself.

Perhaps a way to get this started is to compile the current union of tags folks have used today and then start a draft page in this repo, or as a Gist to iterate and refine the "curated tags" list from that.

I'd be happy to take a first crack at that if you'd like.

**Comment by DanielJDufour** [comment]
Hi, @IanLee1521 .  Great points.  I agree that it sounds like a lot of work though and I'd recommend on holding off.  I'm skeptical that we, on the code.gov side, would have the resources to sustain a tag curation effort.
